components/collector/tests/source_collectors/harbor/test_security_warnings.py

```diff
@@ -21,25 +21,28 @@ def setUp(self) -> None:
 
     async def collect_data(self):
         """Collect the (fake) Harbor contents."""
+        project = self.PROJECT_NAME
         return await self.collect(
             get_request_json_side_effect=[
-                [dict(name=self.PROJECT_NAME)],
-                [dict(name=self.REPO_NAME)],
+                [{"name": project}],
+                [{"name": self.REPO_NAME}],
                 [
-                    dict(
-                        addition_links=dict(
-                            vulnerabilities=dict(
-                                href=f"/api/v2.0/projects/{self.PROJECT_NAME}/repositories/tianon%252Fpostgres-upgrade/"
+                    {
+                        "addition_links": {
+                            "vulnerabilities": {
+                                "href": f"/api/v2.0/projects/{project}/repositories/tianon%252Fpostgres-upgrade/"
```

Comment:
Why this change, was the line too long? The previous line was more clear

Reply:
What color would you like? We have black, black, and black ;-)